doc/tools/docgram/orderedGrammar

```diff
@@ -2380,7 +2380,8 @@ atomic_tac2pat: [
 ]
 
 tac2mode: [
-| ltac2_expr [ "." | "..." ]      (* ltac2 plugin *)
+| OPT ( toplevel_selector ":" ) ltac2_expr [ "." | "..." ]      (* ltac2 plugin *)
+| "par" ":" ltac2_expr [ "." | "..." ]      (* ltac2 plugin *)
```

Comment:
par is included in toplevel_selector, not sure how to remove this redundancy.

Reply:
You can add something like:

```
tac2mode: [
| DELETE "par" ":" ltac2_expr [ "." | "..." ]
]
```

in `doc/tools/docgram/common.edit_mlg`. Untested but there is documentation that should help confirm or fix this.

Reply:
@SkySkimmer do you want to tweak it before I merge?